instaclustr/structs.go

```diff
@@ -21,57 +21,58 @@ type OmitEmptyBool struct {
-	SaslJaasConfig                      string `json:"sasl.jaas.config,omitempty" mapstructure:"sasl_jaas_config,omitempty"`
-	BootstrapServers                    string `json:"bootstrap.servers,omitempty" mapstructure:"bootstrap_servers,omitempty"`
-	Truststore                          string `json:"truststore,omitempty" mapstructure:"truststore,omitempty"`
-	RedisMasterNodes                    int    `json:"masterNodes,omitempty" mapstructure:"master_nodes,omitempty"`
-	RedisReplicaNodes                   int    `json:"replicaNodes,omitempty" mapstructure:"replica_nodes,omitempty"`
-	RedisPasswordAuth                   *bool  `json:"passwordAuth,omitempty" mapstructure:"password_auth,omitempty"`
-	DedicatedZookeeper                  *bool  `json:"dedicatedZookeeper,omitempty" mapstructure:"dedicated_zookeeper,omitempty"`
-	ZookeeperNodeSize                   string `json:"zookeeperNodeSize,omitempty" mapstructure:"zookeeper_node_size,omitempty"`
-	ZookeeperNodeCount                  int    `json:"zookeeperNodeCount,omitempty" mapstructure:"zookeeper_node_count,omitempty"`
-	PostgresqlNodeCount                 int    `json:"postgresqlNodeCount,omitempty" mapstructure:"postgresql_node_count,omitempty"`
-	PostgresqlReplicationMode           string `json:"replicationMode,omitempty" mapstructure:"replication_mode,omitempty"`
-	PostgresqlSynchronousModeStrict     *bool  `json:"synchronousModeStrict,omitempty" mapstructure:"synchronous_mode_strict,omitempty"`
-	PgBouncerPoolMode                   string `json:"poolMode,omitempty" mapstructure:"pool_mode,omitempty"`
-	CadenceAdvancedVisibility           *bool  `json:"useAdvancedVisibility,omitempty" mapstructure:"advanced_visibility,omitempty"`
-	CadenceUseWebAuth                   *bool  `json:"useCadenceWebAuth,omitempty" mapstructure:"use_cadence_web_auth,omitempty"`
-	CadenceTargetCassandraDataCentreID  string `json:"targetCassandraCdcId,omitempty" mapstructure:"target_cassandra_data_centre_id,omitempty"`
-	CadenceTargetCassandraVPCType       string `json:"targetCassandraVpcType,omitempty" mapstructure:"target_cassandra_vpc_type,omitempty"`
-	CadenceTargetOpensearchDataCentreID string `json:"targetOpenSearchCdcId,omitempty" mapstructure:"target_opensearch_data_centre_id,omitempty"`
-	CadenceTargetOpensearchVPCType      string `json:"targetOpenSearchVpcType,omitempty" mapstructure:"target_opensearch_vpc_type,omitempty"`
-	CadenceTargetKafkaDataCentreID      string `json:"targetKafkaCdcId,omitempty" mapstructure:"target_kafka_data_centre_id,omitempty"`
-	CadenceTargetKafkaVPCType           string `json:"targetKafkaVpcType,omitempty" mapstructure:"target_kafka_vpc_type,omitempty"`
-	AdvertisedHostName                  string `json:"advertisedHostName,omitempty" mapstructure:"advertised_host_name,omitempty"`
+	AuthnAuthz                           *bool  `json:"authnAuthz,omitempty" mapstructure:"auth_n_authz,omitempty"`
```

Comment:
Im assuming these all update just because you indented them to fit the "IntegrateRestProxyWithSchemaRegistry"

Reply:
Yeah, that's right.